common/api/imodeljs-common.api.md

```diff
@@ -808,11 +806,18 @@ export class Cartographic implements LatLongAndHeight {
     static parametricLatitudeFromGeodeticLatitude(geodeticLatitude: number): number;
     static scalePointToGeodeticSurface(point: Point3d, result?: Point3d): Point3d | undefined;
     toEcef(result?: Point3d): Point3d;
-    // (undocumented)
-    toJSON(): LatLongAndHeight;
+    toJSON(): CartographicProps;
     toString(): string;
     }
 
+// @public
+export interface CartographicProps {
+    height?: number;
+    isDegrees?: boolean;

```

Comment:
Instead of isDegrees, should these be AngleProps?

Reply:
@mgooding Interesting suggestion - would clean it up a bit.  Only downside is the mysterious 'if it's a number, it is degrees' statement in the AngleProps documentation.  But as long as mostly AngleProps is { radians: value } or { degrees: value }, that could be good...

Reply:
@mgooding Changed to use AngleProps.  I tried to change the codebase everywhere necessary.